src/parser/warrior/protection/CONFIG.js

```diff
@@ -14,16 +14,15 @@ export default {
   // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
   contributors: [Abelito75],
   // The WoW client patch this spec was last updated to be fully compatible with.
-  patchCompatibility: '7.3.5',
+  patchCompatibility: '8.0',
```

Comment:
Why 8.0 and not 8.2?

Reply:
As long as `isSupported` is false, they'll be shown the Warning before being able to see the analysis. Patch compatibility means that the user can at least trust the analysis is updated, even if it's incomplete.